datadog/threadstats/base.py

```diff
@@ -285,13 +298,16 @@ def get_user(user_id):
             finally:
                 stats.histogram('user.query.time', time.time() - start)
         """
+
```

Comment:
Was it done by an automatic formatter❔

Reply:
oh yes. It is to separate the inner function definition from the rest of the function code. PyCharm formatted that automatically